src/wp-includes/link-template.php

```diff
@@ -1814,6 +1814,7 @@ function get_next_post( $in_same_term = false, $excluded_terms = '', $taxonomy =
  * Can either be next or previous post.
  *
  * @since 2.5.0
+ * @since 6.9.1 Adds deterministic fallback for sort clause if not modified by a filter.
```

Comment:
Should probably include the since changes for 6.0.0 that yours truly also missed.

Reply:
What are they? 😄

Reply:
Maybe something like this?

```suggestion
 * @since 6.9.0 Introduce deterministic fallback based in IDs to account for date collisions.
 * @since 6.9.1 Remove deterministic fallback for sites modifying the WHERE clause via a filter.
 *                       See https://core.trac.wordpress.org/ticket/64390
```